### PR TITLE
fix(storybook): delay snapshot tests past animation time for consiste…

### PR DESCRIPTION
**Pull request by ckifer**
…nt results

We take snapshots using chromatic, these give us visual diffs of our storybook stories. We currently get different diffs every time on stories with animation because of animation duration. 

## Description

Add a delay (for now) as a common parameter to the stories with animation.

See https://www.chromatic.com/docs/delay

Eventually we may try to disable animations only for chromatic snapshots. See: https://www.chromatic.com/docs/animations#javascript-animations

- some random auto-formatting changes (prettier did it)

## Related Issue

N/A - didn't make an issue

## Motivation and Context

Consistent snapshots

## How Has This Been Tested?

Built storybook, ran storybook. Confirmed on my local chromatic build that animation stories completed in totality.

## Screenshots (if appropriate)
image

## Types of changes

- [x] testing tool bug fix
- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)

## Checklist:

- [x] My code follows the code style of this project.
- [ ] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.
- [ ] I have added tests to cover my changes.
- [x] All new and existing tests passed.

### CODE REVIEW

**Comment by ckifer** [comment]
@nikolasrieble  - this should help us have less chromatic changes per build and be able to trust the snapshots a bit more

File: storybook/stories/API/props/StoryParameters.ts [hunk 1]
```diff
@@ -0,0 +1,6 @@
+import type { Parameters } from '@storybook/react';
+
+export const SHARED_STORY_PARAMS: Parameters = {
+  // Sets a delay of 1550ms before each story to give them time to animate (recharts animation default is 1500ms)
+  chromatic: { delay: 1550 },
+};
```
Comment (nikolasrieble) [inline storybook/stories/API/props/StoryParameters.ts:6]: Should these not be https://storybook.js.org/docs/react/writing-stories/parameters#global-parameters ?

**Review by nikolasrieble — APPROVED** [review]
Great, thank you! Left one suggestion / question, but not blocking

File: storybook/stories/API/props/StoryParameters.ts (hunk 1 above)
Comment (ckifer) [reply storybook/stories/API/props/StoryParameters.ts:6]: Lol 🙃 I was looking really hard for that but I couldn't find it. I think this works and is (obviously) easier to maintain. Thanks!

**Comment by ckifer** [comment]
Lets try this change instead... Global delay + a little refactor to TS for the preview file.

Note that not all components have animation/need the delay but this is a good first step

**Pull request merged**